src/process/validation/rules/validateMask.ts

```diff
@@ -90,8 +90,8 @@ export function matchInputMask(value: any, inputMask: any) {
 }
 
 export const shouldValidate = (context: ValidationContext) => {
-    const { component, value } = context;
-    if (!isValidatableComponent(component) || !value) {
+    const { component, value, instance } = context;
+    if ((instance as any)?.skipMaskValidation || !isValidatableComponent(component) || !value) {
```

Comment:
I'm not sure this fully gets us there. While this may cover client-side validation when we release the 5.x renderer (although we should confirm), this change presumes that a renderer instance will be passed into the context; that's generally _not_ what happens when we server-side validate (we don't, like, serialize the instance over the wire, we just have to depend on form and component JSON definitions). I think we need more guidance here, because this ticket seems to have been created for a customer, so my questions are:

1. skipMaskValidation seems to [only appear in the Time component](https://github.com/formio/formio.js/blob/2e982e4fedb759a1da87efeba47c9f21ce54aab8/src/components/time/Time.js#L93) and it always returns true. Is there any other place that it appears that I'm missing?
2. Does the customer rely on this value in their evaluation contexts?
3. Does the customer rely on this value in one or more custom components? 

Let's discuss in dev support if possible